dev-requirements.txt

```diff
@@ -1,10 +1,10 @@
 coveralls==3.3.1
 eralchemy==1.5.0
-faker==18.11.2
 mock==5.0.2
 nose>=1.3.1
 pandas>=0.23.4
 pre-commit==3.8.0
+pygraphviz==1.14
```

Comment:
Done, @sea-kelp.

Reply:
Sorry, just noticed that the eralchemy README actually mentions that we need to specify extras:
```
$ pip install eralchemy[graphviz]
```
https://pypi.org/project/eralchemy/

Reply:
This should be addressed now.